pointcloud/basic_octree_utils.go

```diff
@@ -7,12 +7,16 @@ import (
 	"github.com/pkg/errors"
 )
 
+// emptyProb is assigned to nodes who have no value specified.
+var emptyProb = int(math.Inf(-1))
```

Comment:
This is not good practice; as there is no "infinity" representation as an int, this is a very-obfuscated way to say "the smallest possible int", with a bonus of potential floating point/architectural issues on the side.

Consider instead:
```
const emptyProb = math.MinInt
```

Reply:
I like this approach, if we are staying with `int` representation here, we should not do weird casting tricks. I second the above suggestion.